tests/e2e/cucumber/features/app-provider/officeSuites.feature

```diff
@@ -37,6 +38,41 @@ Feature: Integrate with online office suites like Collabora and OnlyOffice
     Then "Anonymous" should not be able to edit content of following resource
       | resource         | type         | content                     |
       | OpenDocument.odt | OpenDocument | Edited OpenDocument Content |
+
```

Comment:
```suggestion
```
Is this necessary?

Reply:
Just to separate the concern of sharing between the "Anonymous" user and "real/logged-in" user.
Should I remove it?